convert2rhel/checks.py

```diff
@@ -447,3 +456,59 @@ def _bad_kernel_substring(kernel_release):
         )
         return True
     return False
+
+
+def check_package_updates():
+    """Ensure that the system packages installed are up-to-date."""
+    logger.task("Prepare: Checking if the packages are up-to-date")
+
+    # Get a total count of how many packages is left to update
+    total_packages = get_total_packages_to_update()
+
+    if total_packages > 0:
+        logger.warning(
+            "The system has %s packages to be updated. "
+            "To proceed with the conversion, update the packages on your system by executing the following step:\n\n"
+            "Run: yum update -y\n" % total_packages
+        )
+    else:
+        logger.info("System is up-to-date.")
+
+
+def is_loaded_kernel_latest():
+    """Check if the loaded kernel is behind or of the same version as in yum repos."""
+    logger.task("Prepare: Checking if the loaded kernel version is the most recent.")
+
+    # For OL/CentOS 8 the `kernel` is just a meta package, instead, we check for `kernel-core`.
+    # But for 6 and 7 releases, the correct way to check is using `kernel`.
+    package_to_check = "kernel-core" if system_info.version.major >= 8 else "kernel"
+
+    # The latest kernel version on repo
+    repoquery_output, _ = run_subprocess('repoquery -q --qf "%{BUILDTIME}\\t%{VERSION}-%{RELEASE}" ' + package_to_check)
```

Comment:
```suggestion
    repoquery_output, _ = run_subprocess('repoquery --quiet --qf "%{BUILDTIME}\\t%{VERSION}-%{RELEASE}" ' + package_to_check)
```
The `-q` means `no-op for rpmquery compatibility` on RHEL 6 - i.e. it does nothing.